libvirt.go

```diff
@@ -294,6 +300,28 @@ func (d *Driver) Create() error {
 	return d.Start()
 }
 
+func createImage(src, dst string) error {
+	start := time.Now()
+	defer func() {
+		log.Debugf("image creation took %s", time.Since(start).String())
+	}()
+	if _, err := exec.LookPath("qemu-img"); err != nil {
+		return mcnutils.CopyFile(src, dst)
+	}
```

Comment:
We could have the `CopyFile` fallback trigger on `qemu-img create ...` failures too, rather than only doing this when `qemu-img` is missing?